Dockerfiles/Dockerfile.frontend

```diff
@@ -5,4 +5,5 @@ ENV REACT_APP_NOMOCK=on
 WORKDIR /web/frontend
 COPY ../web/frontend .
 RUN npm install
-ENTRYPOINT ["npm", "start"]
+ENTRYPOINT ["npm"]
+CMD ["start"]
```

Comment:
This starts the frontend in HTTPS mode with a self-signed certificate. Do we really want that? I propose either one of:

1. `CMD ["start-dev"]`
2. changing the `start` target in `packages.json` to not include the HTTPS=true

Reply:
I went with No.2

Reply:
Oh my - I wrote too fast w/o thinking.

2. in fact doesn't work if you want to test locally with Gaspar. And for the `build` target, we can even remove the `HTTPS=` completely, as it doesn't do anything.

So:
- remove the `HTTPS=false` in `packages.json`
- add a `start-https` target with `HTTPS=true`

Reply:
hmm, so I guess the default should be with `https=true` since it's meant to run a similar to production environment, and when you use `docker-compose.debug.yaml` then it switches you to the `https=false` version?

so:
 - We keep start as https:true
 - start-dev would have https=false

Reply:
@ineiti I forgot a mention